pkg/api/api.proto

```diff
@@ -212,14 +205,11 @@ message GetTrialsReply {
     repeated Trial trials = 1;
 }
 
-message RunTrialRequest {
-    string study_id = 1;
-    string trial_id = 2;
-    string runtime = 3;
-    WorkerConfig worker_config = 4;
+message CreateWorkerReauest {
```

Comment:
Filed https://github.com/kubeflow/katib/issues/150
Would be good to document the API.

Reply:
It can store the worker info to DB and generate workerId by CreateWorker rpc. 
I would change rename the RPC to `RegisterWorker`.
> Would be good to document the API.

OK.

Reply:
Thanks for the explanation. I don't think this needs to change in this PR.  I'd suggest filing an issue if you think its worth fixing.